src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,10 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
```

Comment:
The component doesn't validate the 'todo' prop. It's a good practice to validate your props using PropTypes. This can help prevent bugs and help other developers understand your component's API.